pyproject.toml

```diff
@@ -20,16 +20,16 @@ Jinja2 = "==3.1.1"
 MarkupSafe = "==2.1.1"
 packaging = "==21.3"
 prompt-toolkit = "==3.0.28"
-python = "^3.7"
-PyYAML = "==6.0"
+python = "==3.9.10"
+PyYAML = "==6.0.1"
 termcolor = "==1.1.0"
 
 [tool.poetry.dev-dependencies]
 coverage = "^6.3.2"
 httpretty = "^1.1.4"
 ipdb = "^0.13.9"
 mock = "^4.0.3"
-pyinstaller = "^4.5.1"
+pyinstaller = "^4.6"
```

Comment:
`pyinstaller` needed to be bumped due to a [known issue](https://github.com/pyinstaller/pyinstaller/issues/6325) with a failing sanity check.